static/js/directives/index.js

```diff
@@ -10,5 +10,6 @@
   require('./modal');
   require('./report-image');
   require('./sender');
+  require('./enter');
```

Comment:
We keep these files in alphabetical order mostly to reduce the chance of merge conflicts - if everyone adds to the end you're guaranteed to conflict!